CODEOWNERS

```diff
@@ -7,24 +7,33 @@
 # Order is important. The last matching pattern has the most precedence.
 # So if a pull request only touches javascript files, only these owners
 # will be requested to review.
-test/*                       @jajanusz
-src/math/*                   @singalsu
-src/arch/xtensa/*            @tlauda
-src/platform/*               @tlauda
-*.sh                         @jajanusz
-src/include/sof/preproc*.h   @mwierzbix
-test/cmocka/include/*        @mwierzbix
-src/audio/src*               @singalsu
-src/audio/eq*                @singalsu
-src/audio/fir*               @singalsu
-src/audio/iir*               @singalsu
-src/audio/tone.c             @singalsu
-src/drivers/intel/cavs/dmic.c @singalsu
-src/include/sof/dmic.h       @singalsu
-src/host/*		     @ranj063
-src/include/host/*	     @ranj063
-*.am                         @jajanusz
-*.ac                         @jajanusz
+test/*					@jajanusz
+src/math/*				@singalsu
+src/arch/xtensa/*			@tlauda
+src/platform/*				@tlauda
+*.sh					@jajanusz
+src/include/sof/preproc*.h		@mwierzbix
+test/cmocka/include/*			@mwierzbix
+src/audio/src*				@singalsu
+src/audio/eq*				@singalsu
+src/audio/fir*				@singalsu
+src/audio/iir*				@singalsu
+src/audio/tone.c			@singalsu
+src/drivers/intel/cavs/dmic.c		@singalsu
+src/include/sof/dmic.h			@singalsu
+src/host/*				@ranj063
+src/include/host/*			@ranj063
+tools/topology/*			@ranj063
```

Comment:
you can add this only because if you own the whole topology folder then it means you own all its sub-folder also?

Reply:
@keyonjie  good to know. let me modify it. Thanks!